packages/volto-slate/news/5452.bugfix

```diff
@@ -0,0 +1 @@
+Fixed markup shortcut for heading and blockquote @kHAPPY2004
```

Comment:
What exactly was broken, and how was it fixed? Please include some details, such as what was the original broken behavior and what is the new fixed behavior. These change log entries appear in documentation upon release, so they should be helpful to a developer looking to fix a broken behavior that they observed.